src/tokenizer.rs

```diff
@@ -374,6 +377,17 @@ impl<'a> Tokenizer<'a> {
                 '+' => self.consume_and_return(chars, Token::Plus),
                 '*' => self.consume_and_return(chars, Token::Mult),
                 '%' => self.consume_and_return(chars, Token::Mod),
+                '|' => {
+                    chars.next(); // consume the '|'
+                    match chars.peek() {
+                        Some('|') => self.consume_and_return(chars, Token::StringConcat),
+                        // a regular '/' operator
+                        _ => Err(TokenizerError(format!(
+                            "Tokenizer Error at Line: {}, Col: {}",
```

Comment:
Made some modifications to the error message.

Probably, it would be useful though to support `|` too.

Reply:
Also, another improvement would be to include a simple function which adds line/column information in a consistent way.

Reply:
> Probably, it would be useful though to support `|` too.

Yes, I see no reason not to.

> include a simple function which adds line/column information in a consistent way.

Agreed. I see you filed #179 about that.

Reply:
Yes :)